atomics/T1072/T1072.yaml

```diff
@@ -66,3 +66,42 @@ atomic_tests:
     elevation_required: true
     command: |
       "%PROGRAMFILES(x86)%/#{PDQ_Deploy_exe}"
+
+- name: Deploy 7-Zip Using Chocolatey
+  auto_generated_guid: 2169e8b0-2ee7-44cb-8a6e-d816a5db7d8a
+  description: |
+    An adversary may use Chocolatey to remotely deploy the 7-Zip file archiver utility.
+  supported_platforms:
+  - windows
+  input_arguments:
+    chocolatey_installer:
```

Comment:
Installer names used during prereqs typically don't have a compelling reason to be configurable. Instead I'd prefer the installer name to be hardcoded in the prereq commands and the input argument removed. thx